### ISSUE TITLE
Invariant violation: requireNativeComponent: "RCTNodePlayer" was not found in the UIManager.

### ISSUE THREAD

**Issue by tashanemclean** [NodeMedia/react-native-nodemediaclient#101]
Has this component been taken out of the UIManager? I've spent the last 4hours on react-native github searching for this component but I am unable to find it anywhere. should this repo be considered outdated then?

**Comment by danielfitzpatrick314** [comment]
same issue

**Comment by Moni-h** [comment]
I have same problem and get this error
RCTNodePlayer was not found in the UIManager.
How i can solve it?

**Comment by tashanemclean** [comment]
I believe as of react native 60+ the RTCNodePlayer cannot/ should not be accessed from the UIManager. One solution is use a older version of react native.

**Comment by Moni-h** [comment]
@tashanemclean i use expo how can i fix it?

**Comment by tashanemclean** [comment]
You can use an older version of expo that uses react native 59>. Or you can use react native INIT to build your project. If your goal is to add live streaming to your application then I would recommend using something like bambuster. https://bambuser.com/

**Comment by Moni-h** [comment]
@tashanemclean 
Want to start rtsp streaming with native camera.
Is this lib suport rtsp streaming?

**Comment by tashanemclean** [comment]
Yes, If aiming for a very low latency, near real-time, e.g. as in video conferencing, most implementations use UDP based protocols like RTP. Higher level protocols such as RTSP and WebRTC are all based on RTP, as are many direct video calling systems like FaceTime.

**Comment by Moni-h** [comment]
@tashanemclean thanks much
Where i can find usage of this lib, or some example

**Comment by tashanemclean** [comment]
No problem. You can review their docs here. https://bambuser.com/docs/

**Comment by Moni-h** [comment]
@tashanemclean it`s work with bambuser server or i can use my server?

**Comment by tashanemclean** [comment]
When you integrate the bambuster SDK into your application’s frontend, bambuster handles the streaming on their server. I highly recommend reviewing their documentation as they provide more detailed explanation of their service.

**Comment by tashanemclean** [comment]
Also, it might be a little distasteful to continue discussing a service other than NodeMediaClient on this thread. Please check out the aforementioned documentation!

**Comment by Moni-h** [comment]
ok
